src/FunderMaps.WebApi/Controllers/Report/IncidentController.cs

```diff
@@ -91,41 +96,24 @@ public async Task<IActionResult> GetAllAsync([FromQuery] PaginationModel paginat
         /// <summary>
         ///     Post a new incident to the backend.
         /// </summary>
-        /// <remarks>
-        ///     This call can be made anonymous or as a user.
-        /// </remarks>
         /// <param name="input"><see cref="IncidentDto"/></param>
         /// <returns><see cref="OkObjectResult"/></returns>
-        [AllowAnonymous]
         [HttpPost]
         [ProducesResponseType((int) HttpStatusCode.OK)]
         public async Task<IActionResult> CreateAsync([FromBody] IncidentDto input)
         {
             // Map.
             var incident = _mapper.Map<Incident>(input);
 
-            // Process the anonymity of the request.
-            if (_authManager.IsSignedIn(User))
-            {
-                var sessionUser = await _authManager.GetUserAsync(User);
-                var sessionOrganization = await _authManager.GetOrganizationAsync(User);
-
-                incident.Meta = new
-                {
-                    SessionUser = sessionUser.Id,
-                    sessionOrganization = sessionOrganization.Id,
-                    Gateway = Constants.IncidentGateway,
-                };
-            }
-            else
+            var sessionUser = await _authManager.GetUserAsync(User);
+            var sessionOrganization = await _authManager.GetOrganizationAsync(User);
+
+            incident.Meta = new
```

Comment:
Is it a good idea to store this as a json object of any format? Shouldn't we be a bit more consistent with this?

Reply:
There is no mention of JSON in the controller, could have been XML/TOML/CONF

Reply:
Regardless of the format, do we really want an anonymous object here?

Reply:
No do don't. Can use a local type